maidsafe/safe_browser
# fix(Tabs):Tab sizes consistent

Resolves #521 

 Steps to test the resolution.

QA:
The easiest way to test this PR would be to:
- The browser 
- Open several tabs
-Check If tabs sizes remain consistent
